docs/config.md

```diff
@@ -13,12 +13,136 @@ SPDX-License-Identifier: Apache-2.0
+                          sd: "010203" # Slice Differentiator (3 bytes hex string, range: 000000~FFFFFF)
+              defaultAmf:
+                hostName: amf # Host name of AMF
+                ipAddr: # AMF IP address
+                port: 38412 # AMF port
+            gnb2:
+              n2IpAddr: # gNB N2 interface IP address used to connect to AMF
+              n2Port: 9488 # gNB N2 Port used to connect to AMF
+              n3IpAddr: 192.168.251.6 # gNB N3 interface IP address used to connect to UPF
+              n3Port: 2152 # gNB N3 Port used to connect to UPF
+              name: gnb2 # gNB name that uniquely identify a gNB within application
+              globalRanId:
+                plmnId:
+                  mcc: 208 # Mobile Country Code (3 digits string, digit: 0~9)
+                  mnc: 93 # Mobile Network Code (2 or 3 digits string, digit: 0~9)
+                gNbId:
+                  bitLength: 24
+                  gNBValue: "000112" # gNB identifier (3 bytes hex string, range: 000000~FFFFFF)
+              supportedTaList:
+              - tac: "000001" # Tracking Area Code (3 bytes hex string, range: 000000~FFFFFF)
+                broadcastPlmnList:
+                  - plmnId:
+                      mcc: 208
+                      mnc: 93
+                    taiSliceSupportList:
+                        - sst: 1 # Slice/Service Type (uinteger, range: 0~255)
+                          sd: "010203" # Slice Differentiator (3 bytes hex string, range: 000000~FFFFFF)
+              defaultAmf:
+                hostName: amf # Host name of AMF
+                ipAddr: # AMF IP address
+                port: 38412 # AMF port
 
 - **profiles:**
 	List of test/simulation profiles. Each item in the list holds configuration specific to a profile.
 	Each profile executes required set of systems procedures for the configured set of IMSI's.
 	Each profile has enable field and if set to true then that profile is executed.
-        
+
+- **Getting gNBSim golang profile**
+
+       config:
+         gnbsim:
+           goProfile:
+             enable: true #enable/disable golang profile in gnbsim
+             port: 5000        
+
+- **Run gNBSim with single Interface or multi interface**
+
+       config:
+         gnbsim:
+           yamlCfgFiles:
+             gnb.conf:
+               configuration:
+                   singleInterface: false #default false i.e. multiInterface. Works well for AIAB
+
+- **CustomProfiles**
+    List of custom profiles. Each item in the list holds configuration specific to a customProfile.
+    Each custom profile has enable field and if set to true then that custom profile is executed.
+    Support of Custom Profiles: User can now define your own profile. New profile can be
+    created by using existing baseline procedure. Example of custom profile can be found here.
+    Check customProfiles in `gNBSim config <https://github.com/omec-project/gnbsim/blob/main/config/gnbsim.yaml>_`
```

Comment:
The links are following the RST format but they should be converted to MD format
```suggestion
    Check customProfiles in [gNBSim config](../config/gnbsim.yaml)
```